menas/ui/components/login/loginDetail.controller.js

```diff
@@ -45,6 +45,10 @@ sap.ui.define([
         this.handleMaster();
       }, this);
 
+      ConfigRestClient.getEnvironmentName()
+        .then( sEnvironmentName => sap.ui.getCore().getModel().setProperty("/menasEnvironment", sEnvironmentName) )
+        .fail(console.log("Failed to get Environment variable"));
```

Comment:
```suggestion
        .fail(console.log("Failed to get Environment name"));
```
lol, same as in the other controller

Reply:
Well, I forgot I have two of those 😑